config/manager/manager.yaml

```diff
@@ -25,13 +25,6 @@ spec:
         command:
         - /moco-controller
         image: ghcr.io/cybozu-go/moco:latest
-        resources:
-          limits:
-            cpu: 4000m
-            memory: 2Gi
-          requests:
```

Comment:
Pods w/o resource requests have the lowest QoS priority (BestEffort) and should be avoided in general.
https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/

I'd suggest having `resources.requests.memory=256Mi`